### DISCUSSION TITLE
ggml_new_tensor_impl: not enough space in the scratch memory

### DISCUSSION THREAD

**Discussion by resford** [ggml-org/whisper.cpp#670] [General]
First off thank you for your work! This is really exciting stuff and I look forward to messing around with it. I downloaded a sample wav file and only get to about 2 minutes of transcription before I receive the following error message. 

ggml_new_tensor_impl: not enough space in the scratch memory

I am on a relatively new M1 mac so I am assuming that shouldnt be an issue. I have tried tiny.en up to the large and have had similar results. Any idea why this might be the case?

**Comment by ggerganov** [comment]
#671
